src/components/common/walletConnect/utils.tsx

```diff
@@ -0,0 +1,38 @@
+import { ReactComponent as IconWallet1 } from 'assets/icons/wallet-1.svg';
+import { ReactComponent as IconWallet2 } from 'assets/icons/wallet-2.svg';
+import { ReactComponent as IconWallet3 } from 'assets/icons/wallet-3.svg';
+import { ReactComponent as IconWallet4 } from 'assets/icons/wallet-4.svg';
+import { ReactComponent as IconWallet5 } from 'assets/icons/wallet-5.svg';
+import { ReactComponent as IconWallet6 } from 'assets/icons/wallet-6.svg';
+import { i18n } from 'libs/translations';
+
+export const getProductDescriptionItems = () => {
```

Comment:
utils isn't a good name for this file

Reply:
amm, is data.tsx sounds better ?

Reply:
@zavelevsky can u take a look at this again pls?

Reply:
Changed to items.tsx